### PR TITLE
Added miscelaneous resource about different topics

### PR DESCRIPTION
**Pull request by terranigmark**
## What does this PR do?
Add resource(s)

### Description
Added new resources about programming languages, web development, and Git.
### Why is this valuable (or not)?
The content consists of short ebooks made of hand-written notes which are used to improve learning experiences about tech concepts.
### How do we know it's really free?
Access to the notes is free.
### For book lists, is it a book? For course lists, is it a course? etc.
Books.
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-es.md [hunk 1]
```diff
@@ -110,6 +110,7 @@
 * [Docker](https://github.com/brunocascio/docker-espanol#docker) (Github)
 * [El camino a un mejor programador](http://emanchado.github.io/camino-mejor-programador/downloads/camino_2013-01-19_0688b6e.html) (HTML) [(PDF, ePub)](http://emanchado.github.io/camino-mejor-programador/)
 * [Introducción a Docker](https://www.rediris.es/tecniris/archie/doc//TECNIRIS47-1b.pdf) (PDF)
+* [Los Apuntes de Majo](https://losapuntesdemajo.vercel.app) - Majo Ledesma (HTML)
```
Comment (davorpa) [inline books/free-programming-books-es.md:113]: Hey @majoledesma ya entraste en nuestra lista.

:information_source: Links inside points to Google Drive

Comment (terranigmark) [reply books/free-programming-books-es.md:113]: Acabo de ver que en la guía de contribución no se admiten recursos en Google Drive, en ese caso este PR se cerraría por ahora ¿correcto? 🤔

**Comment by eshellman** [comment]
why do you think this is a book?

File: books/free-programming-books-es.md (hunk 1 above)
Comment (davorpa) [reply books/free-programming-books-es.md:113]: En este caso creo que GDrive sería válido ya que es como Majo lo mantiene

**Review by davorpa — APPROVED** [review]

File: books/free-programming-books-es.md (hunk 1 above)
Comment (davorpa) [reply books/free-programming-books-es.md]: Si puedes responder a la pregunta de Eric estaría genial

**Pull request merged**